src/Build/Graph/ProjectGraph.cs

```diff
@@ -243,6 +297,92 @@ private static ImmutableList<string> ExpandDefaultTargets(ProjectInstance projec
             return targets;
         }
 
+        /// <summary>
+        /// Gets the effective global properties for a project reference.
+        /// </summary>
+        /// <remarks>
+        /// The behavior of this method should match the logic in Microsoft.Common.CurrentVersion.targets and the MSBuild task.
+        /// </remarks>
+        private static PropertyDictionary<ProjectPropertyInstance> GetProjectReferenceGlobalProperties(ProjectItemInstance projectReference, PropertyDictionary<ProjectPropertyInstance> requesterGlobalProperties)
+        {
+            string propertiesString = projectReference.GetMetadataValue(PropertiesMetadataName);
+            string additionalPropertiesString = projectReference.GetMetadataValue(AdditionalPropertiesMetadataName);
+            string undefinePropertiesString = projectReference.GetMetadataValue(UndefinePropertiesMetadataName);
+            string globalPropertiesToRemoveString = projectReference.GetMetadataValue(GlobalPropertiesToRemoveMetadataName);
+
+            // TODO: Mimic AssignProjectConfiguration's behavior for determining the values for these.
```

Comment:
Bug for this? Or just a rollup "solutions in graph" one?

Reply:
Sure, I'll open a new issue for it and throw it in some future Project